q.plugin.zsh

```diff
@@ -7,6 +7,29 @@ ZSH_HIGHLIGHT_REGEXP+=('\bq.*\b' 'fg=green,bold')
 ZSH_HIGHLIGHT_REGEXP+=('\bQ.*\b' 'fg=green,bold')
 ZSH_HIGHLIGHT_REGEXP+=('\bU.*\b' 'fg=green,bold')
 
+# Check for a custom Q command
+echo $Q_SET
+echo $Q_RUN
+echo $Q_UNSET
+
+# Check if Q_SET is defined
+if [[ -z $Q_SET ]]; then
+    echo 'Using default set'
```

Comment:
And here! ;)

Reply:
You want those ones gone too?

Reply:
Yeah! Don't need to tell the user everytime! ;)